articles/quickstart/backend/java-spring-security5/02-using.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Using your API
+description: This tutorial will show you how to use your API.
+budicon: 500
+topics:
+    - quickstart
+    - backend
+    - java
+    - spring-security
+contentType: tutorial
+useCase: quickstart
+---
+
+<%= include('../_includes/_api_using') %>
```

Comment:
Wait.. Isn't this what you already explained at the end of 01-authorization file?

Reply:
Yes, see comment above.

Reply:
Updated `_api_using.md` to include instructions for obtaining test tokens, and removed the Testing instructions from the Authorization article.